Contest_API.md

```diff
@@ -1032,9 +1032,9 @@ Properties of problem objects:
 | ordinal           | number    | A unique number that determines the order the problems, e.g. on the scoreboard.
 | rgb               | string ?  | Hexadecimal RGB value of problem color as specified in [HTML hexadecimal colors](https://en.wikipedia.org/wiki/Web_colors#Hex_triplet), e.g. `#AC00FF` or `#fff`.
 | color             | string ?  | Human readable color description associated to the RGB value.
-| time\_limit       | number ?  | Time limit in seconds per test data set (i.e. per single run). Should be an integer multiple of `0.001`.
-| test\_data\_count | integer ? | Number of test data sets.
-| max_score         | number ?  | Maximum expected score, although teams may score higher in some cases. Typically used to indicate scoreboard cell color in scoring contests. Required iff contest:scoreboard_type is `score`.
+| time\_limit       | number    | Time limit in seconds per test data set (i.e. per single run). Should be an integer multiple of `0.001`.
+| test\_data\_count | integer   | Number of test data sets.
+| max_score         | number    | Maximum expected score, although teams may score higher in some cases. Typically used to indicate scoreboard cell color in scoring contests. Required iff contest:scoreboard_type is `score`.
```

Comment:
I guess this should be declared in `/access` only if (but not necessarily if) the scoreboard type is "score". If it is declared I guess it should be present for *each* problem, so not be nullable, but maybe @niemela can judge better.

Reply:
Given the vagueness built-in ("although teams may score higher in some cases") I think we could make this non-nullable.